definitions/stateless_apps.md

```diff
@@ -1,19 +1,27 @@
 ---
 title: Stateless Apps
 status: Feedback Appreciated
-category: technology
+category: Technology
 ---
 
 ## Stateless Apps
 
 ### What it is
 
-A stateless application doesn’t save any client session (state) data on the server where the application lives. Each session is carried out as if it was the first time and responses are not dependent upon data from a previous session and provides functionality to use print services, CDN (Content Delivery Network) or the Web Servers in order to process every short-term request. For example, someone is searching a question in the search engine and pressed the Enter button. In case if the searching operation gets interrupted or closed due to some reason, you have to start a new one as there is no saved data for your previous request.
+- A Stateless application doesn’t save any client session (state) data on the server where the application lives.
+- Each session is carried out as if it was the first time.
+- Responses are not dependent upon data from a previous session.
+- Provides functionality to use print services, CDN (Content Delivery Network) or the Web Servers in order to process every short-term request.
+- For example, when you use a Coke machine, you’re the one who maintains the state. After buying a drink, the machine doesn’t remember what kind of drink you purchased, but you do because you’re holding it in your hand. In this respect, it’s more accurate to say that a stateless application externalizes its state rather than maintaining it on the server. 
```

Comment:
No other entry is using bullet points. Please convert this section to a paragraph or two. Try and keep the whole section to no more than 6 sentences.